column/api/controller/run_controller.py

```diff
@@ -0,0 +1,33 @@
+import uuid
+
+from flask import request
+from flask_restful import Resource
+
+from column.api import backend
```

Comment:
I don't know what your coding standard is, but maybe combine these two imports for simplicity?